rustls-postquantum/examples/client.rs

```diff
@@ -0,0 +1,58 @@
+//! This is the simplest possible client using rustls-postquantum, based on
```

Comment:
Would you mind also adding a `cargo run --locked --bin rustls-postquantum` to the `.github/workflows/daily-tests.yml`'s `example-tests` section? It could go alongside the `simpleclient`, `limitedclient`, etc steps.

Reply:
Good point, have added this, with a `grep` at the end to check this basically works.

Reply:
Hah, that was surprisingly annoying. It seems some cloudflare POPs serve different pages for pq.cloudflareresearch.com - in the UK I get a static HTML page, but CI in the US gets javascript 🤨 

So now this hits the API endpoint that the javascript targets.

Reply:
Tricksy CDNs and their tricksy games :roll_eyes: